yogstation/code/game/gamemodes/gangs/gang_items.dm

```diff
@@ -308,6 +307,37 @@
 	permeability_coefficient = 0.01
 	clothing_flags = NOSLIP
 
+/datum/gang_item/equipment/stashbox
+	name = "Stashbox"
+	id = "stashbox"
+	cost = 10
+	item_path = /obj/item/stashbox
+
+/datum/gang_item/equipment/stashbox/spawn_item(mob/living/carbon/user, datum/team/gang/gang, obj/item/gangtool/gangtool)
+	if(item_path)
```

Comment:
Early return
Check if it's false and return TRUE, no need for if / else

Reply:
Its a  copy+paste of how the original does it on line 30

Reply:
doesn't mean you shouldn't change it :)